From b15c4c4b24c894b3c8d444466110c881c35525e2 Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org> 
Date: Wed, 14 Nov 2018 15:35:37 +0100 
Subject: [PATCH 4/5] gentoo: fix update_package_sources function

Method is meant to update packages repository, not to update the system.
---

diff --git a/cloudinit/distros/gentoo.py b/cloudinit/distros/gentoo.py
index 702cda8a..37217fe4 100644
--- a/cloudinit/distros/gentoo.py
+++ b/cloudinit/distros/gentoo.py
@@ -246,7 +246,7 @@ class Distro(distros.Distro):
         self._runner.run(
             "update-sources",
             self.package_command,
-            ["-u", "world"],
+            ["--sync"],
             freq=PER_INSTANCE,
         )
 
